templates/sponsors/sponsorship_detail.html

```diff
@@ -0,0 +1,63 @@
+{% extends "users/base.html" %}
+{% load humanize pipeline %}
+
+{% block head %}
+  {% stylesheet 'font-awesome' %}
+{% endblock %}
+
+{% block page_title %}
+  {{ sponsorship.sponsor.name }} Sponsorship Application | {{ SITE_INFO.site_name }}
+{% endblock %}
+
+{% block body_attributes %}class="psf signup default-page"{% endblock %}
+
+
+{% block main-nav_attributes %}psf-navigation{% endblock %}
+
+{% block user_content %}
+  <h1>{{ sponsorship.sponsor.name }} Sponsorship Application</h1>
+
+  <h3>Application Data</h3>
+  <ul>
+    <li>
+      <b>Status:</b> {{ sponsorship.get_status_display }}
+    </li>
+    <li>
+      <b>Application date:</b> {{ sponsorship.applied_on|default_if_none:"---" }}
+    </li>
+    <li>
+      <b>Approval date:</b> {{ sponsorship.approved_on|default_if_none:"---" }}
+    </li>
+    <li>
+      <b>Start date:</b> {{ sponsorship.start_date|default_if_none:"---" }}
+    </li>
+    <li>
+      <b>End date:</b> {{ sponsorship.end_date|default_if_none:"---" }}
+    </li>
+    {% if sponsorship.finalized_on %}
+    <li>
+      <b>Finalized date:</b> {{ sponsorship.finalized_on }}
+    </li>
+    {% endif %}
+    <li>
+      <b>Level:</b> {{ sponsorship.level_name }}
+    </li>
+    <li>
+      <b>Agreed sponsorship fee:</b> ${{ sponsorship.sponsorship_fee|intcomma }}
```

Comment:
After applying, this is currently showing the package price even if the benefits have been modified. We should show a similar "TBD" text as we do on the confirmation page.